algosdk/atomic_transaction_composer.py

```diff
@@ -506,13 +507,15 @@ def execute(
                         raw_value=raw_value,
                         return_value=return_value,
                         decode_error=decode_error,
+                        tx_info=client.pending_transaction_info(tx_id),
```

Comment:
For completenes, making requests to `PendingTransactionInfo/{TXID}` even for non-app transactions.

Reply:
Do you mean even for methods that return void? Because that's what this change does.

Also, any functions that might raise errors (`client.pending_transaction_info` in this case) must be placed in the try block to catch the error, so I's suggest moving the start of that try block up to contain the entire loop body if you want to keep this change

Reply:
👍